plenum/test/batching_3pc/test_basic_batching.py

```diff
@@ -98,3 +105,63 @@ def rejectingMethod(self, req):
 
     for i, node in enumerate(txnPoolNodeSet):
         node.doDynamicValidation = origMethods[i]
+
+
+def test_msg_max_length_check_node_to_node(tconf,
```

Comment:
The test should belong to another module. This module is about 3PC batches, not transport batches.

Reply:
Done